### PR TITLE
PS-7441 Document RocksDB variable "rocksdb_max_compaction_history" an…

### PR DESCRIPTION
**Pull request by patrickbirch**
…d deprecate "strict_collation_check" (8.0)

modified:   source/myrocks/variables.rst

### CODE REVIEW

File: doc/source/myrocks/variables.rst [hunk 1]
```diff
@@ -2295,13 +2311,15 @@ Specifies whether to include checksums when writing index or table records.
...
 .. variable:: rocksdb_strict_collation_check
-
+  
   :cli: ``--rocksdb-strict-collation-check``
   :dyn: Yes
   :scope: Global
   :vartype: Boolean
   :default: ``ON``
 
+This variable is considered **deprecated** in version 8.0.21-12.
```
Comment (inikep) [inline doc/source/myrocks/variables.rst]: It will be deprecated from 8.0.23

File: doc/source/myrocks/variables.rst [hunk 2]
```diff
@@ -2314,6 +2332,8 @@ Enabled by default.
   :vartype: String
   :default:
 
+This variable is considered **deprecated** in version 8.0.21-12.
```
Comment (inikep) [inline doc/source/myrocks/variables.rst]: It will be deprecated from 8.0.23

**Review by inikep — CHANGES_REQUESTED** [review]

File: doc/source/myrocks/variables.rst (hunk 2 above)
Comment (patrickbirch) [reply doc/source/myrocks/variables.rst]: fixed.

File: doc/source/myrocks/variables.rst (hunk 1 above)
Comment (patrickbirch) [reply doc/source/myrocks/variables.rst]: fixed.

**Review by inikep — APPROVED** [review]
LGTM

**Pull request merged**